tests/Drupal/Arrays/ArrayUnitTest.inc

```diff
@@ -10,7 +10,7 @@ $array = [
   'inline' => [],
   'inline3' => ['one', 'two', 'three'],
   'inline_long_ok' => ['one', 'two', 'three', 'four', 'five'],
-  'inline_long_nok' => ['one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight'],
```

Comment:
we should not change the existing test case, please add a new one.

Reply:
OK will do.

Reply:
I have reverted the changed test lines, and added new instead. This also showed where a previous line had 101 chars but was allowed to pass, hence the change in getErrorList()